### PR TITLE
Introduce Sublime Text plugin LSP-promql

### PR DESCRIPTION
**Pull request by nevill**
It's already been in the official package control channel. 🎉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,3 +95,9 @@ With Vim, currently only PromQL queries inside YAML files work without additiona
 #### Debugging
 
 The Vim command `:YcmDebugInfo` gives status information and points to logfiles.
+
+### Sublime Text 3
+
+With ST3, you need to install package `LSP`, `LSP-promsql` via `Package Control`. Follow the installation instruction of [LSP-promql](https://github.com/nevill/lsp-promql#installation).
```
Comment (slrtbtfs) [inline README.md:101]:
```suggestion
With ST3, you need to install package `LSP`, `LSP-promql` via `Package Control`. Follow the installation instruction of [LSP-promql](https://github.com/nevill/lsp-promql#installation).
```
I think this is a typo.

File: README.md [hunk 2]
```diff
@@ -95,3 +95,9 @@ With Vim, currently only PromQL queries inside YAML files work without additiona
...
+
+Now, you can start editing any file with `.promql` as suffix.
```
Comment (slrtbtfs) [inline README.md:103]: Do you plan to support `.yaml` files, too?

**Comment by slrtbtfs** [comment]
Thanks for this PR! Great to see more editors offering PromQL support.

File: README.md (hunk 2 above)
Comment (nevill) [reply README.md:103]: Do you mean embeded promql in YAML? I don't know how to do that right now. The syntax highlight in ST3 can do, but not the LSP plugin. That's why I emphasized to edit a file with suffix`.promql`.

Comment (slrtbtfs) [reply README.md:103]: You can simply enable the language server for all YAML files.

The language server then takes care of searching for PromQL queries inside that YAML file.

At least on VS Code this still works well when the YAML language server is enabled, too, and the performance overhead introduced by the PromQL language server is small compared to the resources taken by VS Code and the YAML language server.

Comment (nevill) [reply README.md]: Now it works within YAML, as you can see [there](https://github.com/nevill/lsp-promql#usage).
It takes me some time until I realize that `promql-langserver` only works with specific nodes in YAML, basically in a Prometheus config YAML.

**Comment by slrtbtfs** [comment]
Thanks!

**Pull request merged**